services/galley/src/Galley/Intra/Client.hs

```diff
@@ -183,6 +187,28 @@ getClientByKeyPackageRef ref = do
     then Just <$> parseResponse (mkError status502 "server-error") r
     else pure Nothing
 
+-- | Calls 'Brig.API.Internal.getConvIdByKeyPackageRef'.
+getConvIdByKeyPackageRef :: KeyPackageRef -> App (Maybe (Qualified ConvId))
+getConvIdByKeyPackageRef ref = do
+  r <-
+    call Brig $
+      method GET
+        . paths ["i", "mls", "key-packages", toByteString' $ toUrlPiece ref, "conversation"]
+        . expectStatus (flip elem [200, 404])
+  if statusCode (responseStatus r) == 200
+    then Just <$> parseResponse (ErrorCall . LText.unpack) r
```

Comment:
Why `ErrorCall` here? In other analogous places we've used `mkError status502 "server-error"` instead.